src/modules/Profile/components/FavoriteList.tsx

```diff
@@ -1,15 +1,33 @@
 import { Empty } from "antd";
 import { useTranslation } from "react-i18next";
 
-import { ProfileTitle } from "../Profile.styled";
+import { useAppSelector } from "hooks/useRedux";
+import { RootState } from "redux/store";
+import { ILaptop } from "types/laptop.model";
+import { FavoriteListContainer, ProfileTitle } from "../Profile.styled";
+import FavoriteCard from "./FavoriteCard";
 
 const FavoriteList = () => {
   const { t } = useTranslation();
+  const { favoriteList } = useAppSelector((state: RootState) => state.favorite);
 
   return (
     <>
       <ProfileTitle>{t("profile:favorite")}</ProfileTitle>
-      <Empty image={Empty.PRESENTED_IMAGE_SIMPLE} />;
+      {favoriteList.length ? (
```

Comment:
Update if loading

Reply:
It have 2 function is adding to favorite list and removing from favorite list. So I think it has not loading